src/app/work-item/work-item.service.ts

```diff
@@ -57,6 +57,24 @@ export class WorkItemService {
       .catch(this.handleError);
   }
 
+ getWorkItemTypes(): Promise<any[]> {
+    this.logger.log('WorkItemService using url ' + this.workItemTypeUrl);
+    let url = this.workItemTypeUrl;
+    return this.http
+      .get(url, { headers: this.headers })
+      .toPromise()
+      .then(response => {
+        if (process.env.ENV == 'inmemory') {
```

Comment:
Why was this broken into the longer "if" statement version? I appears to be an unnecessary change.